src/main/java/domain/gambler/Players.java

```diff
@@ -0,0 +1,39 @@
+package domain.gambler;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Players {
+    private static final int MAXIMUM_PLAYER_COUNT = 7;
+    private static final int MINIMUM_PLAYER_COUNT = 1;
+    private static final String NULL_EXCEPTION_MESSAGE = "Null exception.";
+    private static final String OUT_OF_RANGE_EXCEPTION_MESSAGE = "Player number out of range exception. (1,7)";
+    private final List<Player> players;
+
+    public Players(Map<Name, Money> playerInfo) {
+        validatePlayerInfo(playerInfo);
+        this.players = Collections.unmodifiableList(playerInfo.keySet().stream()
+                .map(playerName -> new Player(playerName, playerInfo.get(playerName)))
+                .collect(Collectors.toList()));
```

Comment:
```suggestion
        this.players = playerInfo.entrySet()
                .stream()
                .map(Player::new)
                .collect(toUnmodifiableList());
```

와 형태로 구현한다면 가독성이 상승합니다.
메서드 레퍼런스와 스트림에 대해 공부해보아요.

Reply:
감사합니다:) toUnmodifiableList()가 JAVA10부터 지원을 하는군요! JAVA8의 방식으로 수정했습다!